src/Workspaces/Core/Portable/Storage/CloudCache/ICloudCacheStorageServiceFactory.cs

```diff
@@ -6,8 +6,8 @@
 
 namespace Microsoft.CodeAnalysis.Storage.CloudCache
 {
-    internal interface ICloudCacheStorageServiceFactory : IWorkspaceService
+    internal interface ICloudCacheStorageServiceProvider : IWorkspaceService
```

Comment:
rename file to `ICloudCacheStorageServiceProvider`

Reply:
done.